src/openms/source/CHEMISTRY/RibonucleotideDB.cpp

```diff
@@ -43,21 +43,34 @@ using namespace std;
 
 namespace OpenMS
 {
-  RibonucleotideDB::RibonucleotideDB():
-    max_code_length_(0)
+
+  RibonucleotideDB* RibonucleotideDB::getInstance()
   {
-    readFromFile_("CHEMISTRY/Modomics.tsv");
-    readFromFile_("CHEMISTRY/Custom_RNA_modifications.tsv");
+    static RibonucleotideDB* db_ = RibonucleotideDB::initializeRibonucleotideDB();
```

Comment:
this heap object will never be destroyed now (not even when the app closes). Is that intended?